lib/gringotts/gateways/cams.ex

```diff
@@ -1,320 +1,359 @@
 defmodule Gringotts.Gateways.Cams do
-  @moduledoc ~S"""
-    A module for working with the Cams payment gateway.
-    
-    You can test gateway operations in [CAMS API TEST MODE](https://secure.centralams.com).
-    Test it using these crediantials **username:** `testintegrationc`, **password:** `password9`,
-    as well as you can find api docs in this test account under **integration** link.
-
-    The following features of CAMS are implemented:
-
-    | Action                       | Method        |
-    | ------                       | ------        |
-    | Authorize                    | `authorize/3` |
-    | Capture                      | `capture/3`   |
-    | Purchase                     | `purchase/3`  |
-    | Refund                       | `refund/3`    |
-    | Cancel                       | `void/2`      |
+  @moduledoc """
+  [CAMS][home] gateway implementation.
+
+  CAMS provides a [sandbox account][dashboard] with documentation under the
+  [`integration` tab][docs]. The login credentials are:
+
+  | Key      | Credentials        |
+  | ------   | --------           |
+  | username | `testintegrationc` |
+  | password | `password9`        |
+
+  The [video tutorials][videos] (on vimeo) are excellent.
+
+  The following features of CAMS are implemented:
+
+  | Action                       | Method        |
+  | ------                       | ------        |
+  | Authorize                    | `authorize/3` |
+  | Capture                      | `capture/3`   |
+  | Purchase                     | `purchase/3`  |
+  | Refund                       | `refund/3`    |
+  | Cancel                       | `void/2`      |
 
   ## The `opts` argument
 
-    Most `Gringotts` API calls accept an optional `Keyword` list `opts` to supply
-    optional arguments for transactions with the Cams gateway. The following keys
-    are supported:
-  
-    | Key                 | Remark | Status          |
-    | ----                | ---    | ----            |
-    | `billing_address`   |        | Not implemented |
-    | `address`           |      	 | Not implemented |
-    | `currency`          |        | **Implemented** |
-    | `order_id`  				|        | Not implemented |
-    | `description`       |        | Not implemented |
-
-    All these keys are being implemented, track progress in
-    [issue #42](https://github.com/aviabird/gringotts/issues/42)!
-
-  ## Configuration parameters for Cams:
-
-    | Config parameter | Cams secret   |
-    | -------          | ----          |
-    | `:username`      | **Username**  |
-    | `:password`      | **Password**  |
-  
+  Most `Gringotts` API calls accept an optional `keyword` list `opts` to supply
+  optional arguments for transactions with the CAMS gateway. The following keys
+  are supported:
+
+  | Key                 | Remark | Status          |
```

Comment:
Please fill this table. AFAICS `billing_address` and `order_id` are already implemented, but there is no explanation what their schema or types are. Either place a link to the official docs or document the schema somewhere.